tensorflow/compiler/xla/runtime/BUILD

```diff
@@ -257,26 +257,15 @@ cc_library(
         "@llvm-project//llvm:OrcJIT",
         "@llvm-project//llvm:Support",
         "@llvm-project//llvm:TransformUtils",
+    ] + if_llvm_aarch64_available([
+        "@llvm-project//llvm:AArch64AsmParser",
+        "@llvm-project//llvm:AArch64CodeGen",
+    ]) + if_llvm_aarch32_available([
+        "@llvm-project//llvm:ARMAsmParser",
+        "@llvm-project//llvm:ARMCodeGen",
+    ]) + if_llvm_x86_available([
         "@llvm-project//llvm:X86AsmParser",
         "@llvm-project//llvm:X86CodeGen",
-    ] + select({
-        "//tensorflow/tsl:arm_any": [
-            "@llvm-project//llvm:AArch64AsmParser",  # fixdeps: keep
-            "@llvm-project//llvm:AArch64CodeGen",  # fixdeps: keep
-        ],
-        "//tensorflow/tsl:linux_ppc64le": [
-            "@llvm-project//llvm:PowerPCAsmParser",  # fixdeps: keep
```

Comment:
Is it purposeful that some of these dependencies were fully removed?

Reply:
No, I must have made a mistake in my rebase. Let me correct that.